Makefile

```diff
@@ -32,10 +32,10 @@ include/hexxer.hpp:
 	curl 'https://raw.githubusercontent.com/dcousens/hexxer/d76f9e526676535fd4c2584a8f84582994c35996/hexxer.h' > $@
 
 include/ranger.hpp:
-	curl 'https://raw.githubusercontent.com/dcousens/ranger/e5b156dd0d8d3de14c2c807a2d7874af0e9cd3e3/ranger.hpp' > $@
+	curl 'https://raw.githubusercontent.com/dcousens/ranger/ddf75d976cb2e1bd4a5b69543b36f0778f90762b/ranger.hpp' > $@
 
 include/serial.hpp:
-	curl 'https://raw.githubusercontent.com/dcousens/ranger/e5b156dd0d8d3de14c2c807a2d7874af0e9cd3e3/serial.hpp' > $@
+	curl 'https://raw.githubusercontent.com/dcousens/ranger/ddf75d976cb2e1bd4a5b69543b36f0778f90762b/serial.hpp' > $@
```

Comment:
44d9037b29a91be0dfce178a8a67314530919e45 is latest HEAD

Reply:
oops. wonder where I copied that commit from... it's nowhere in ranger...